userStoryInventoryChangHF.md

```diff
@@ -39,41 +39,646 @@ This section shows the areas or sources thus far
+
+## 4E: Display Voting Power for DReps on Governance Actions Page
+
+**User Story:** As a DRep connected to GovTool, I want my voting power to be displayed when I look at the governance actions page.
+
+### Acceptance Criteria
+
+- Given that I am a DRep and connected to GovTool.
+- When I look at the governance actions page.
+- Then my voting power is displayed.
+
+## 4F: Ability to Disconnect Wallet and Disable DRep Functionality
+
+**User Story:** As a DRep connected to GovTool, I want the ability to disconnect my wallet from the platform and temporarily disable DRep functionality.
+
+### Acceptance Criteria
+
+- Given that I am a DRep and connected to GovTool and I am on the governance actions page.
+- When I click "Disconnect Wallet".
+- Then my wallet is disconnected and I am redirected to the same page without DRep functionality enabled, including the ability to vote.
+
+### 4G: View Current Vote Count on Governance Action Details Page
+
+**User Story:** As a DRep, I want to be able to view the current vote count for, against, and abstaining on the details page of an individual governance action.
+
+#### Acceptance Criteria
+
+- Given that I am a DRep.
+- When I look at the details page of an individual governance action.
+- Then I can see the current number of votes for, against, and abstaining.
+
+## 4H: Verify That Displayed Governance Actions Have Not Expired
+
+**User Story:** As a user on the governance action page, I want to verify that none of the displayed governance actions have expired, been ratified, or enacted.
+
+### Acceptance Criteria
+
+- Given that I am on the governance action page.
+- When I examine the governance actions displayed.
+- Then none of the governance actions shown on the page have expired, been ratified, or enacted.
+
+## 4I: Count Most Recent Vote in the Same Snapshot
+
+**User Story:** As a DRep, I want my most recent vote to be counted if I submit a different vote for the same transaction within the same snapshot.
+
+### Acceptance Criteria
+
+- Given that I am a DRep and I have already voted on a given governance action.
+- When I submit a different vote for the same transaction within the same snapshot.
+- Then the most recent vote will be counted.
+
+## 4J: Include Metadata Anchor in Voting Transaction
+
+**User Story:** As a user who has chosen how to vote on the UI of a governance action's details, I want to include a metadata anchor when voting and ensure it's included in the resulting transaction.
+
+### Acceptance Criteria
+
+- Given that I have chosen how to vote on the UI of a governance action's details.
+- When I add a metadata anchor to the UI and click the vote button.
+- Then the resulting transaction will include my metadata anchor.
+
+## 5A: Required Information Displayed for Governance Action
+
+**User Story:** As any user, when I look at a governance action it will have the following information displayed so that I can make an informed choice.
```

Comment:
what information?

Reply:
Thanks, The information has been included.